config/kafka-monitor.properties

```diff
@@ -39,8 +39,11 @@
     "zookeeper.connect": "localhost:2181",
     "bootstrap.servers": "localhost:9092",
     "produce.record.delay.ms": 100,
-    "produce.topic.autoTopicCreationEnabled": true,
-    "produce.topic.autoTopicReplicationFactor" : 1,
+    "produce.topic.topicCreationEnabled": true,
+    "topic-management.replicationFactor" : 1,
+    "topic-management.partitionsPerBroker" : 2,
```

Comment:
Change to:
partitionToBrokerRatio (double, use ceil when creating partitions, must be greater than or eq 0).
partitionToBrokerRatioThreshold (just a name change)